tests/templates/test_subroutines/test_select.py

```diff
@@ -743,8 +744,10 @@ def test_no_ops(self, num_controls, partial):
         control = list(range(num_controls))
         work = list(range(num_controls, 2 * num_controls - 1))
 
-        decomp = _select_decomp_unary(ops=[], control=control, work_wires=work, partial=partial)
-        assert decomp == []
+        with qml.tape.QuantumTape() as tape:
+            _select_decomp_unary(ops=[], control=control, work_wires=work, partial=partial)
+
+        assert tape.operations == []
```

Comment:
core told me that we should use queues instead of tapes.
```suggestion
        with qml.queuing.AnnotatedQueue() as q:
            _select_decomp_unary(ops=[], control=control, work_wires=work, partial=partial)

        assert len(q) == 0
```